### PR TITLE
Release 15

### PR DESCRIPTION
**Pull request by mstajdohar**

### CODE REVIEW

**Comment by dblenkus** [comment]
And there are some unnecessary change log entries, like fixing tests, etc.

File: docs/CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,19 @@ Change Log
 All notable changes to this project are documented in this file.
 This project adheres to `Semantic Versioning <http://semver.org/>`_.
 
+==========
```
Comment (dblenkus) [inline docs/CHANGELOG.rst]: There should be 2 empty lines above.

**Comment by mstajdohar** [comment]
> there is inconsistent use of dots at the end of bullet points

Dots are used if there are more then 2 sentences in the bullet line. Is this OK?

**Comment by mstajdohar** [comment]
Should we always use dots?

**Comment by mstajdohar** [comment]
> Few lines in the change log are longer than 79 characters and there is inconsistent use of dots at the end of bullet points.

Fixed

> And there are some unnecessary change log entries, like fixing tests, etc.

Fixed

> There should be 2 empty lines above.

Fixed

**Comment by dblenkus** [comment]
> Should we always use dots?

I think we should never use them.

> Dots are used if there are more then 2 sentences in the bullet line. Is this OK?

I agree if multiple sentences are inevitable. In some cases they can be easily reworded in to a single sentence, e.g. ```Add ``resolwebio/snpeff`` Docker image. The image includes SnpEff (4.3K) tool.```

**Pull request merged**